src/components/CommentTextEntry.jsx

```diff
@@ -1,22 +1,55 @@
+import { useState } from 'react'
 import SendIcon from '@mui/icons-material/Send';
 import InputBase from '@mui/material/InputBase';
 import Paper from '@mui/material/Paper';
 import IconButton from '@mui/material/IconButton';
+import { postComment } from '../network/network';
 
-export default function CommentTextEntry(){
+export default function CommentTextEntry({user, article_id, setComments}){
+    
+    const [comment, setComment] = useState('')
+    const [sent, setSent] = useState(true)
+    
+    const handleSendComment = async ()=>{
+        
+        setSent(false) 
+        const res = await postComment(article_id, user, comment)
+        if (await res.status === 200){
+            setSent(true)
+            setComments((currentComments)=>[res.data, ...currentComments])
+            setComment('')
+        }
+    }
```

Comment:
What happens if something goes wrong while posting?

Reply:
Thought I'd leave this until the 'error handling' ticket but will complete now. The plan is to use the NetworkErrorAlert component which is in the second file in the network directory.